### PR TITLE
Fonts

### PR DESCRIPTION
**Pull request by sam33rdhakal**
Updates
* Adds the app fonts 
* Adds styles according to Zeplin
* Updates the home screen with new style

### CODE REVIEW

File: welcome/src/main/kotlin/edu/artic/welcome/WelcomeExhibitionAdapter.kt [hunk 1]
```diff
@@ -19,7 +19,10 @@ class OnViewAdapter : AutoHolderRecyclerViewAdapter<WelcomeExhibitionCellViewMod
                 .disposedBy(item.viewDisposeBag)
 
         item.exhibitionDate
-                .bindToMain(exhibitionDate.text())
+                .subscribe {
+                    val throughString = context.getString(R.string.through, it)
+                    exhibitionDate.text = throughString
```
Comment (agrosner) [inline welcome/src/main/kotlin/edu/artic/welcome/WelcomeExhibitionAdapter.kt]: intermediate variable not necessary.

File: build.gradle [hunk 2]
```diff
@@ -55,7 +57,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.0-beta03'
+        classpath 'com.android.tools.build:gradle:3.3.0-alpha03'
```
Comment (agrosner) [inline build.gradle]: should we be using an alpha of the build tools? was there a reason for this change?

File: welcome/src/main/res/layout/welcome_tour_summary_cell_layout.xml [hunk 3]
```diff
@@ -1,84 +1,80 @@
...
-    android:layout_height="wrap_content"
-    android:layout_marginEnd="@dimen/marginTwenty">
+    android:layout_height="match_parent">
 
+    <ImageView
+        android:id="@+id/image"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:scaleType="centerCrop"
+        app:layout_constraintDimensionRatio="H,285:177"
```
Comment (agrosner) [inline welcome/src/main/res/layout/welcome_tour_summary_cell_layout.xml]: nice!!!

Comment (raigex) [reply welcome/src/main/res/layout/welcome_tour_summary_cell_layout.xml]: yea we found this and it's amazing

File: build.gradle (hunk 2 above)
Comment (sam33rdhakal) [reply build.gradle]: Nope. It was accidental, removed it.

File: welcome/src/main/kotlin/edu/artic/welcome/WelcomeExhibitionAdapter.kt (hunk 1 above)
Comment (sam33rdhakal) [reply welcome/src/main/kotlin/edu/artic/welcome/WelcomeExhibitionAdapter.kt]: Done.

File: welcome/src/main/res/layout/welcome_tour_summary_cell_layout.xml (hunk 3 above)
Comment (sam33rdhakal) [reply welcome/src/main/res/layout/welcome_tour_summary_cell_layout.xml]: 👍 yea it is amazing.

**Pull request merged**